main.js

```diff
@@ -60,7 +60,18 @@ startTimerButton.addEventListener('click', function() {
 logActivityButton.addEventListener('click', logActivityEvents);
 createNewActivityButton.addEventListener('click', returnToActivityForm);
 
-//Add functions below
+// Add functions below
+(function populatePastActivities() {
+  var keys = Object.keys(localStorage);
+  for (var i = 0; i < keys.length; i++) {
+        pastActivities.push(localStorage.getItem(keys[i]));
```

Comment:
**Suggestion:** I would avoid touching local storage directly here in your `main` file.
You generally want to separate your persistence layer (that's local storage in this case) from the rest of your application because it makes your main logic much easier to think about.

It looks like this statement `localStorage.getItem(keys[i])` is getting some items you've stored there. I'd imagine those are `Activities`. Perhaps this statement can be wrapped in a method and put into `Activity` so you can call something like `activities = Activity.getItems()`.